app/views/prisoners/results.html

```diff
@@ -7,32 +7,22 @@
 {% block content %}
 
 <div class="govuk-grid-row">
-  <div class="govuk-grid-column-two-thirds">
+  <div class="govuk-grid-column-one-half">
     <h1 class="govuk-heading-xl govuk-!-margin-bottom-6">
-      Prisoners search results
+      Search results: prisoners
     </h1>
   </div>
   <div class="govuk-grid-column-full">
-    <!-- <h2 class="govuk-heading-m">Search</h2> -->
-    <form class="form" method="post">
-      <div class="govuk-form-group">
-        <label class="govuk-label" for="width-20">
-          Prisoner number or name
-        </label>
-        <input class="govuk-input govuk-!-width-two-thirds" id="prisonerSearch" name="prisonerSearch" type="text" value="{{ data['prisonerSearch'] }}">
-        <button type="submit" class="govuk-button">
-          Search
-        </button>
-        <p><a href="/advanced-search/" class="govuk-link">Advanced search</a></p>
+    <div class="govuk-grid-row">
+      <div class="govuk-grid-column-one-half">
+        {%- if prisoners.length == 0 -%}
+        <p class="govuk-body-l">Your search for <span class="govuk-!-font-weight-bold">"{{ data['prisonerSearch'] }}"</span> returned no results</p>
+        {%- else -%}
+        <p class="govuk-body-l">1 result containing prisoner number or name <span class="govuk-!-font-weight-bold">"{{ data['prisonerSearch'] }}"</span></p>
```

Comment:
Line should read:
Results containing 'XXXXX' 

(Do we need to say how many??)

Also, in 'Credits recieved' in the table, it's 'received', (e before i, after the c)